Automattic/sensei
# Add mobile layout for learning mode

Fixes #4484 


### Changes proposed in this Pull Request
- Add mobile/responsive layout to course theme:
- Make sidebar a toggle-able menu
- Move lesson actions to sticky bottom footer, hidden when scrolling
- Adjust sizes & spacing a bit

### Testing instructions

* Enable the feature flag: `add_filter( 'sensei_feature_flag_course_theme', '__return_true' );`
* Take out mobile phone
* Open lesson of a course with course theme enabled
* Browse around the course. Make sure sidebar and action buttons are showing correctly



### Screenshot / Video

<img src="https://user-images.githubusercontent.com/176949/145651748-bb2ef915-4fe8-4823-ba6e-1ea402a03e4f.gif" width=240 />
